ClientApp/src/components/AddListing.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { useForm } from "react-hook-form";
```

Comment:
react-hook-form is an outside library that you introduced, therefore it should be added to package.json -> dependencies so that it is installed automatically with other dependencies.

Reply:
the same might be true for axios (if it isn't among dependencies already)